doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -191,6 +191,15 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0
+
+NCSDK-31095: Using the :kconfig:option:`CONFIG_SEGGER_SYSVIEW` Kconfig option will cause the data
+  parameter in the macros :c:macro:`k_fifo_put`, :c:macro:`k_fifo_alloc_put`, :c:macro:`k_lifo_put`
+  and :c:macro:`k_lifo_alloc_put` to be evaluated multiple times. This can i. e. cause problems if
+  the data parameter is a function call incrementing a reference counter.
```

Comment:
```suggestion
NCSDK-31095: Issues with the :kconfig:option:`CONFIG_SEGGER_SYSVIEW` Kconfig option
  Using this Kconfig option causes the data parameter in the macros :c:macro:`k_fifo_put`, :c:macro:`k_fifo_alloc_put`, :c:macro:`k_lifo_put`, and :c:macro:`k_lifo_alloc_put` to be evaluated multiple times.
  This can cause problems if the data parameter is a function call incrementing a reference counter.
```